main.py

```diff
@@ -2,9 +2,57 @@
 
 from db.models import Race, Skill, Player, Guild
 
+import json
+
 
 def main() -> None:
-    pass
+    try:
+        with open("players.json") as file:
+            data = json.load(file)
+    except FileNotFoundError:
+        print("File 'players.json' not found.")
+        return
+    except Exception as e:
+        print(f"An error occurred: {e}")
+        return
```

Comment:
Why don't you want to let those errors raise?

Reply:
I thought it was a good idea

Reply:
It's fine for FileNotFound, but you should not catch all errors using Exception